db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericStatement.java

```diff
@@ -603,6 +604,10 @@ else if (newMergeJoinString.equals("forced"))
                 }
                 cc.setDisablePerParallelTaskJoinCosting(disablePerParallelTaskJoinCosting);
 
+                /**
```

Comment:
I guess we can't quite deprecate that one until we support an alternative that can be used for a transition though, but I agree we need to deprecate this in favour of a single parameter for timestamp precision, and a new way to specify the precision at function level: e.g. `CURRENT TIMESTAMP(9)`

Reply:
Yes, maybe I shouldn't write that until we have an alternative. will remove it.